kong/runloop/plugin_servers/init.lua

```diff
@@ -257,22 +258,28 @@ function get_instance_id(plugin_name, conf)
   return instance_info.id
 end
 
+function reset_instances_for_plugin(plugin_name)
```

Comment:
`reset_instances_by_name` is a better name?

Reply:
note this a cherry-pick

Reply:
aha